server/libs/domain/test/fixtures.ts

```diff
@@ -0,0 +1,42 @@
+import { UserEntity } from '@app/infra';
```

Comment:
is it the typo for the file name?

Reply:
No, it's where we can put all the test data we'll reuse. `AuthUserDto`s seem like a good thing to share across tests.